monitoring/oplog-populator/alerts.yaml

```diff
@@ -48,7 +48,7 @@ groups:
           oplog_populator_acknowledgement_lag_sec_count{job="${oplog_populator_job}",namespace="${namespace}"}[1m]
         ))
         >= ${oplogPopulatorChangeStreamLagThreshold}
-    For: "30s"
+    For: "5m"
```

Comment:
if I understand correctly, `For` clause means prom waits for 5m to trigger the alert after it first encounters the issue in Expr, right?
just out of curiosity, how we choose this duration? some of them are 30s, some are 1m some are 5m

Reply:
Yes that's what it means, but only if the condition persists for that amount of time.
There's no real way of choosing it in this case, as we don't know yet how the feature behaves in prod

Reply:
ok I see, so may adjust these numbers accordingly by how they behave in the prod later right?

Reply:
yes